packages/office-addin-telemetry/README.md

```diff
@@ -0,0 +1,44 @@
+# Office-Addin-telemetry
+This package allows for sending telemetry event and exception data to the selected telemetry infrastructure (e.g. ApplicationInsights).
+
+
+## Command-Line Interface
+* [List](#List)
+* [Off](#Off)
+* [On](#On)
+* [Privacy](#Privacy)
```

Comment:
Privacy should not be a command line interface section.  Please remove this and then just put the Privacy blub after #Office-Addin-Telemetry

Reply:
Use two pounds signs so it is a second-level heading just like the CLI section.